### DISCUSSION TITLE
HBO Family (Latin America)

### DISCUSSION THREAD

**Discussion by ELProductions** [iptv-org/community#775] [Q&A]
### Channel Name

HBO Family (Latin America)

### Channel ID

HBOFamilyLatinAmerica.us

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ELProductions** [comment]
Stream URL please.

**Comment by freearhey** [comment]
The channel has been added to our blocklist due to the claims of the copyright holder: [#16839](https://github.com/iptv-org/iptv/issues/16839)
